app/controllers/items_controller.rb

```diff
@@ -15,8 +15,18 @@ def update
       @item.update(status: params[:disable].to_i)
       redirect_to merchant_items_path(@merchant.id) 
      elsif params[:enable] 
-      @item.update(status: params[:enable].to_i)
       redirect_to merchant_items_path(@merchant.id)
     end 
   end
+
+  def new 
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def create 
+    item = Item.create(name: params[:name], description: params[:description], unit_price: params[:unit_price], merchant_id: params[:merchant_id], status: params[:status])
+    merchant = Merchant.find(params[:merchant_id])
+    item.save 
```

Comment:
You do not need to call `save` on an item when you use the `#create` method. The `#create` method is the `#new` and `#save` method rolled into one. :) I would take that line:29 out.

Reply:
A good refactor opportunity here may also be to create strong params so that you can just call 'item_params' rather than listing it all out? This is probably something that all our controllers should have if we are doing a create or update.

Reply:
Created strong params and deleted the .save :)